src/helpers/configuration.sh

```diff
@@ -24,7 +24,7 @@ fi
 if [[ "$(is_git_town_configured)" == false ]]; then
   echo "Git Town hasn't been configured for this repository."
   echo "Please run 'git town config --setup'."
-  echo -n "Would you like to do that now? [y/n] "
+  echo -n "Would you like to do that now?"
```

Comment:
The user should see `Y/n` somewhere, so that he knows that he has to just hit Enter to run the wizard. Where that happens I don't care.